### ISSUE TITLE
Mark a digital representation as primary

### ISSUE THREAD

**Issue by ivozandhuis** [vangoghworldwide/linkedart#21]
How to model a primary digital image? Other images (eg. the backside) as secondary

**Comment by michielhildebrand** [comment]
I can not find such a thing in Linked Art. Ideally a photograph of the backside should be modeled as such. Thus the artwork has a part which is the rear and this part has a digital representation. I am a bit hesitant to add this as it complicates things even further. Images can then be at three different levels (artwork, parts, technical recordings). At the other hand it makes it unambiguous what is depicted. Any opinion about this?

**Comment by ivozandhuis** [comment]
Steering committee explicitly stated photographs of rears must be part of the platform. I think we need the unambiguousness ...

**Comment by ivozandhuis** [comment]
You're right that we should model the rear as a part and than link the image to that. For the time being we could model exactly one primary image for the userinterface of a website with property crm:P138i_has_representation and all others with the property crm:P65_shows_visual_item. That is at least how I interpret the distinction between 'depiction' and  'surrogate' in [Linked Art](https://linked.art/model/digital/).

eg. 
```
@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
@prefix crm: <http://www.cidoc-crm.org/cidoc-crm/> .

<https://hdl.handle.net/21.12130/collect.13C06907-A4F2-486E-BA9D-7392BD766114>
    a crm:E22_Human-Made_Object ;
    crm:P138i_has_representation <https://blabla/X13968_01.tif> ;
    crm:P65_shows_visual_item <https://blabla.com/X13968_02.tif>, <https://blabla.com/X13968_03.tif> ;
    rdfs:seeAlso <https://vangoghworldwide.org/data/artwork/F121> .
```

**Comment by vangoghworldwide** [comment]
Oh no they changed to digital integration model. access_point is new. The notion of digital surrogates is new. All useful constructs. We should update the VGW model (at same point) to the latest in Linked Art, also the provenance. But this will have impact throughout the whole chain, data suppliers, platform, website.

Adopting the digital surrogates now would make it confusing. I also do not interpret the "depiction" vs "surrogate" as the solution we need here.

Is there not an AAT concept for preferred image? Getty AAT site is down.

**Comment by ivozandhuis** [comment]
Suggestion:
Using the AAT concept 'primary (general designation)' http://vocab.getty.edu/aat/300404450 as an extra has_type to the image next to 'digital image' http://vocab.getty.edu/aat/300215302.
